adapters/beachfront/beachfront.go

```diff
@@ -796,3 +799,11 @@ func getDefaultExtraInfo() ExtraInfo {
 		VideoEndpoint: defaultVideoEndpoint,
 	}
 }
+
+func getBannerImpIDs(bfs []beachfrontSlot) []string {
+	impIDs := []string{}
```

Comment:
Please copy the optimization from GetImpIDs and initialize to the length of `bfs`.

Reply:
Done